aiohttp/web_urldispatcher.py

```diff
@@ -755,6 +756,9 @@ def routes(self):
     def named_resources(self):
         return MappingProxyType(self._named_resources)
 
+    def set_defaults(self, *, allow_head):
```

Comment:
I doubt if we need default for the flag.
It might lead to many confusing cases, e.g. if third party library has something like `register_routes(router, prefix)` method its behavior varies depending on outer flag.